lib/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -15,38 +15,41 @@ import 'package:seeds/navigation/navigation_service.dart';
 import 'package:seeds/screens/explore_screens/explore/components/explore_card.dart';
 import 'package:seeds/screens/explore_screens/explore/interactor/viewmodels/explore_bloc.dart';
 import 'package:seeds/screens/explore_screens/explore/interactor/viewmodels/explore_item.dart';
+import 'package:seeds/screens/explore_screens/explore/interactor/viewmodels/explore_page_command.dart';
 import 'package:url_launcher/url_launcher.dart';
 
 /// Explore SCREEN
 class ExploreScreen extends StatelessWidget {
   final List<ExploreItem> _exploreItems = const [
     ExploreItem(
-      title: 'Invite a Friend',
-      icon:
-          Padding(padding: EdgeInsets.only(left: 6.0), child: CustomPaint(size: Size(40, 40), painter: InvitePerson())),
-      route: Routes.createInvite,
-    ),
-    ExploreItem(title: 'Vouch', icon: CustomPaint(size: Size(35, 41), painter: Vouch()), route: Routes.vouch),
+        title: 'Invite a Friend',
+        icon: Padding(
+            padding: EdgeInsets.only(left: 6.0), child: CustomPaint(size: Size(40, 40), painter: InvitePerson())),
+        onTapEvent: OnExploreCardTapped(Routes.createInvite)),
+    ExploreItem(
+        title: 'Vouch',
+        icon: CustomPaint(size: Size(35, 41), painter: Vouch()),
+        onTapEvent: OnExploreCardTapped(Routes.vouch)),
     ExploreItem(
       title: 'Vote',
       icon: Padding(padding: EdgeInsets.only(right: 6.0), child: CustomPaint(size: Size(40, 40), painter: Vote())),
-      route: Routes.vote,
```

Comment:
i updated to pass an event instead of a route, sometimes we dont want to navigate to routes

Reply:
Sounds good the I was thinkig the same but I tried to make a constant list. Maybe we will end using a mutable one;